docs/cloud/integrations/grafana.md

```diff
@@ -0,0 +1,1011 @@
+
+```
+Steampipe service is running:
+Database:
+  Host(s):            localhost, 127.0.0.1, 192.168.29.204
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+```
+
+## Getting started
+
+[Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) is a open source interactive data-visualization platform that runs on the cloud, or in a container, or the desktop. To use Grafana you will need to sign up and create an account. Here, we will use the desktop version.
+
+With Grafana up and running, you can access it from the browser through `http://localhost:3000`. Click on `Data Sources` under the configuration option, then click `Add Data Source`, select `PostgreSQL` and enter the connection details. Since there is no field for port number, it should be added with the host as `Host:Port` and set the TLS/SSL mode to `Require`. Then click `Save & Test` to test your connection.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/grafana-connection-success.png" />
+</div>
+
+The plugins and it's tables can be accessed through the `Explore` tab and the `Edit SQL` option can be used to customize the queries.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/grafana-explore-tables.png" />
+</div>
+
+The data can be previewed in a `Table format` or `Time series`. Here we see the AWS EC2 instances listed in a table format.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/grafana-table-format-data-preview.png" />
+</div>
+
+## Dashborad to monitor resources and cost
+
+Grafana uses panels as the building blocks of a dashboard that consists of queries, graphs, aliases, etc. We'll focus here on creating a dashboard to monitor AWS services and Cost. To being click on `Dashboards` and create a new dashboard. On the empty dashboard click `Add panel` to create a panel and select `aws_ebs_volume_metric_read_ops_daily` from the query builder, then click `Apply`. Now add panels for `aws_ec2_instance_metric_cpu_utilization_hourly`, `aws_s3_bucket_by_region` and `aws_vpc_by_region`. Similarly paste the query into a new panel to monitor the top 10 monthly costs by service.
+
+```
+select
+  service,
+  sum(unblended_cost_amount)::numeric::money as sum,
+  avg(unblended_cost_amount)::numeric::money as average
+from
+  aws_cost_by_service_monthly
+group by
+  service
+order by
+  average desc
+limit 10;
+```
+Grafana gives a wide range of graphics that you can use to represent the data in each panel. Now with the created panels we have our AWS Monitoring dashboard ready.
```

Comment:
```suggestion
Grafana provides a variety of chart types. Here we use A for X, B for Y, and C for Z.  
```

Reply:
where A|B|C are chart types and X|Y|X are metrics.

Reply:
updated the text with the suggested format.